### ISSUE TITLE
Faucet Another Testnet Network

### ISSUE THREAD

**Issue by nmluthfi** [AlmostEfficient/FaucetBot#9]
Can we fork this to use it on another testnet chain? If, so what are the changes needed?

**Comment by AlmostEfficient** [comment]
Yeah, don't need to do anything except change RPC URL to whatever testnet - the code is agnostic
